mongo/integration/unified/operation.go

```diff
@@ -97,6 +97,8 @@ func (op *operation) run(ctx context.Context, loopDone <-chan struct{}) (*operat
 		return executeListCollections(ctx, op)
 	case "listCollectionNames":
 		return executeListCollectionNames(ctx, op)
+	case "rename":
+		return executeRenameCollection(ctx, op)
```

Comment:
Since `operation.object` is a collection, I suggest moving this to below "Collection operations".